bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/dao/impl/BahmniPatientDaoImplLuceneIT.java

```diff
@@ -16,18 +17,19 @@
 import static junit.framework.Assert.assertNull;
 import static junit.framework.Assert.assertTrue;
 
+@Ignore
```

Comment:
any reason this is ignored? Also try to put a reason why this is ignored @Ignore("For so and so ... ")

Reply:
@angshu This requires raising PR to PatientIdentifier.class for adding @sortField for "identifier" field. Updated the details in the below card which we will pick as part of this sprint.
https://bahmni.atlassian.net/browse/BAH-1888